ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_isis.schema.yml

```
@@ -44,6 +44,136 @@ keys:
                   - str
                 default: 10000
                 description: Delay in milliseconds.
+      set_overload_bit:
+        type: dict
+        keys:
+          enabled:
+            type: bool
+          on_startup:
+            type: dict
+            keys:
+              delay:
+                type: int
+                convert_types:
+                  - str
+                description: Number of seconds.
+              wait_for_bgp:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                  timeout:
+                    type: int
+                    convert_types:
+                      - str
+                    description: Number of seconds.
+      authentication:
+        type: dict
+        keys:
+          both:
+            type: dict
+            description: Authentication settings for level-1 and level-2. 'both' takes precedence over 'level_1' and 'level_2' settings.
+            keys:
+              key_type:
+                type: str
+                convert_types:
+                  - int
+                valid_values:
+                  - '0'
+                  - '7'
+                  - '8a'
+                description: Configure authentication key type. Default key_id is 0.
+              key:
+                type: str
+                description: Password string.
+              key_id:
```

This should be some work in plural since it is a list.
```suggestion
              key_ids:
```